src/components/users/users.service.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const usersRepository = require('./users.repository');
+
+function create(name) {
+  return usersRepository.create(name);
+}
+
+async function findAll() {
+  const [...users] = await usersRepository.findAll();
```

Comment:
why do we need it?
`const users = ...`

Reply:
I can check if users exist without using users.length